hunsri/EwMM-ProjectCS
# XP system and first rewards

Added:
- rank 0 to 3
- reward for rank 1
- barricade for the door "level 2" and for reward for rank 1

Test:
Part 1
- open MainMenu scene
- play the scene
- look at the rank board
- change the canvas of the rank board to the reward canvas
- you can see that the first item is locked
- change back to the rank canvas to the reward canvas

Part 2
- open in the hierarchy MainMenu/MainRoom/Adder
- in the inspector add small number like 4 to "Xp" and check "Rank Need To Be Changed"
- press as often "Rank Need To Be Changed" till you reach rank 1
- know change the canvas of the rank board to the reward canvas
- now you can see that the first item is  unlocked
- the player can now start level 2... if it's implemented (spoiler: it's not) °-°
- you can add more XP to the rank till it reached rank 3 (which is the highest rank for now)
